homeassistant/components/modbus/base_platform.py

```diff
@@ -115,7 +115,7 @@ async def async_update(self, now: datetime | None = None) -> None:
     def async_run(self) -> None:
         """Remote start entity."""
         self.async_hold(update=False)
-        self._cancel_call = async_call_later(self.hass, 1, self.async_update)
+        async_call_later(self.hass, timedelta(milliseconds=100), self.async_update)
```

Comment:
Why remove assigning this to `self._cancel_call` ?

Reply:
Done